src/docs/guides/database-migration-guide.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Migrating from Legacy Plugins to Database Services
+---
+
+We've introduced a more secure and flexible way to deploy databases on Railway.  The purpose of this guide is to provide an overview of why we are making these changes, what they mean for you, and how you can migrate.
+
+## What is Changing?
+### Legacy Plugins
+In the early stages of Railway, we introduced `plugins`, a foundational part of our offering. They were databases deployed from Docker images with a fixed version, no data persistence, and no way to customize the database to your needs.
```

Comment:
Big nit. But the code formatting of plugins looks a bit strange for non-code things
```suggestion
In the early stages of Railway, we introduced "Plugins", a foundational part of our offering. They were databases deployed from Docker images with a fixed version, no data persistence, and no way to customize the database to your needs.
```